scriptworker/config.py

```diff
@@ -34,6 +35,10 @@
 _VALUE_UNDEFINED_MESSAGE = "{path} {key} needs to be defined!"
 
 
+class UnknownProduct(Exception):
+    """The configured `cot_product` doesn't have a value for a given key."""
```

Comment:
Hm, does this belong in `scriptworker.exceptions`? Might be a child of `CoTError`.

Reply:
Sure, I hadn't noticed it.  I don't think it should be a child of CoTError, since it happens on worker startup, not as part of verifying a task. I've renamed it `ConfigError`.